src/wallet/sqlite.cpp

```diff
@@ -170,7 +170,16 @@ void SQLiteDatabase::Open(const char* mode)
 
 bool SQLiteDatabase::Rewrite(const char* skip)
 {
-    return false;
+    while (true) {
```

Comment:
nit: maybe add an upper limit to avoid hanging the thread in case something went wrong?

Reply:
I think it's fine

Reply:
Removed the loop